src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/LocationController.kt

```diff
@@ -27,4 +33,17 @@ class LocationController(private val locationService: LocationService) {
                        @ApiParam(value = "The group name", required = true) @PathVariable("name") name: String): List<Location>
     = locationService.getCellLocationsForGroup(agencyId, name)
 
+  @GetMapping("/cellsWithCapacity/{agencyId}/{group}")
+  @ApiOperation(value = "List of cells by group at agency location which have capacity.", notes = "List of cells  by group at agency location which have capacity.", nickname = "getCellsWithCapacityForGroup")
+  @ApiResponses(value = [
+    ApiResponse(code = 200, message = "OK", response = CellWithAttributes::class, responseContainer = "List"),
+    ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse::class, responseContainer = "List"),
```

Comment:
I don't think you need the responseContainer for the errors. Might be worth checking.

Reply:
Looks like you don't, have cleaned up this controller.